consolidate/process_consolidated.go

```diff
@@ -205,6 +212,11 @@ func convertAddress(cosmosAddress string) (string, error) {
 }
 
 func skip(address string) bool {
+	// skip ibc escrow address
+	if ibcEscrowAddress[address] {
+		return true
```

Comment:
```suggestion
		// return true
```

our goal is not to skip them but to lock them